zerver/lib/markdown/__init__.py

```diff
@@ -2078,6 +2079,35 @@ def handleMatch(  # type: ignore[override] # supertype incompatible with supersu
         return None, None, None
 
 
+class EscapePostprocessor(markdown.postprocessors.UnescapePostprocessor):
```

Comment:
I think this should not be renamed, as this still performs the operation that upstream calls “unescape”; it just also fixes a bug by adding a missing escape operation.